scripts/run_alerts_tests.sh

```
@@ -0,0 +1,35 @@
+    echo "promtool could not be found"
+    exit 1
+fi
+
+mkdir -p observability/rules/tests/rules
+
+# For each file under observability/rules directory, create a temp yaml file and run the tests under observability/rules/tests directory
+for file in observability/rules/*.yml
+do
+  # Create temp rule file under observability/rules/tests directory
+  cp $file observability/rules/tests/rules/
+  yq eval -i '.groups = .spec.groups | del(.apiVersion, .kind, .metadata, .groups[].params, .spec)' observability/rules/tests/rules/$(basename $file)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)